### DISCUSSION TITLE
Is there a horizontal scrolling issue with tables?

### DISCUSSION THREAD

**Discussion by LemonPie22** [filamentphp/filament#10565] [Help] [answered]
I've discovered on my application that, when viewing a table on mobile view, it's impossible (or at least hard) to scroll the table horizontally. When I try to swipe horizontally the entire panel moves. I've found the same issue on the demo.

Looking at https://demo.filamentphp.com/shop/products I see a table as shown here:

<img width="620" alt="Screenshot 2023-12-29 at 22 51 19" src="https://github.com/filamentphp/filament/assets/107488019/c843681c-ecc3-4633-bdbc-5cf061e7840e">

When I try to scroll to the right this happens:

<img width="457" alt="Screenshot 2023-12-29 at 22 53 12" src="https://github.com/filamentphp/filament/assets/107488019/7cdffffb-18fc-481e-bc71-6b3d74ddc71c">

**Comment by royalmediaco** [comment]
It impacts vertical scrolling also. Go to https://demo.filamentphp.com/shop/categories, switch to a mobile device and scroll down. There's a bunch of extra space at the bottom and to the right.

I think this is a bug that was introduced in 3.1.23. I don't know the cause or the solution.

**Reply by wolfdave** [reply]
Same here. But only when there is a widgets or other fields above.

**Comment by LemonPie22** [comment, marked as answer]
This has been fixed in v3.1.35.

**Reply by wolfdave** [reply]
Thank you very much.
